sycl/test-e2e/AddressSanitizer/out-of-bounds/USM/large_group_size.cpp

```diff
@@ -0,0 +1,26 @@
+// REQUIRES: linux, cpu
```

Comment:
Our tests can run on gpu now.

```suggestion
// REQUIRES: linux
```